docs/flink_processor.md

```diff
@@ -6,12 +6,22 @@ sections we describe how to use the Flink processor.
 ## Deployment Mode
 The Flink processor runs the Flink job in one of the following mode:
 
+- Cli Mode
```

Comment:
How about naming it `Command Line mode`?

And `Session Mode` -> `Session mode` etc.